EzMap.Domain/Models/EntityBase.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Runtime.InteropServices.JavaScript;
+
+namespace EzMap.Domain.Models;
+
+public class EntityBase<T> 
+{
+    public virtual T Id { get; set; }
+    
+    public Guid CreatedBy { get; set; }
+    public DateTime CreatedDate { get; set; }
+
+    public Guid? LastModifiedBy { get; set; }
+
+    public DateTime? LastModifiedDate { get; set; }
+    
+    public DateTime? DeletedDate { get; set; }
+
+    public Guid? DeletedBy { get; set; }
```

Comment:
Currently all these properties are not populated when saving entities to database. Can reference littleviet repo if unsure how to do it.